docs/admin/general-upgrade-guidelines.rst

```diff
@@ -0,0 +1,73 @@
+.. highlight:: sh
+
+.. _general_upgrade_guidelines:
+
+==========================
+General upgrade guidelines
+==========================
+
+Before kicking off an upgrade, there is a set of best practices to ensure the best outcome. Below you may find the five fundamental steps to prepare for an upgrade.
```

Comment:
>  five fundamental steps to prepare for an upgrade

Isn't it just four? 1) Acknowledge breaking changes, 2) Set up a test environment, 3) Back up and plan recovery, 4) Define a rollback plan.